### PR TITLE
MM-42030: Migrate Threads RHS tour tip to tippy based tour tip

### PR DESCRIPTION
**Pull request by codedsun**
#### Ticket Link

https://github.com/mattermost/mattermost-server/issues/19802

#### Related Pull Requests

- Has server changes (please link here)
- Has mobile changes (please link here)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by AshishDhama** [comment]
@codedsun Could you please fix the broken test.

File: components/crt_tour/crt_threads_pane_tutorial_tip/crt_threads_pane_tutorial_tip.tsx [hunk 1]
```diff
@@ -25,18 +31,46 @@ const CRTThreadsPaneTutorialTip = () => {
...
-            step={Constants.CrtThreadPaneSteps.THREADS_PANE_POPOVER}
-            tutorialCategory={Preferences.CRT_THREAD_PANE_STEP}
-            screen={screen}
-            overlayClass='tip-overlay--threads-pane'
-            punchOut={useMeasurePunchoutsDeprecated(['rhsContainer'], [])}
-            autoTour={true}
+            handleDismiss={onDismiss}
+            handleNext={onDismiss}
+            interactivePunchOut={true}
+            nextBtn={nextBtn()}
```
Comment (saturninoabril) [inline components/crt_tour/crt_threads_pane_tutorial_tip/crt_threads_pane_tutorial_tip.tsx:73]: @codedsun Looks like this should be,
```suggestion
            nextBtn={nextBtn}
```

**Comment by AshishDhama** [comment]
@codedsun Were you able to add the patch?

**Comment by AshishDhama** [comment]
@jfrerich @jgilliam17 This pr is ready for review

**Review by jfrerich — APPROVED** [review]
LGTM

**Review by AshishDhama — APPROVED** [review]
LGTM!

**Comment by AshishDhama** [comment]
@jgilliam17 Gentle reminder on this one.

**Comment by jgilliam17** [comment]
Thanks @codedsun and also @saturninoabril for fixing the pipeline and running the e2es.
There are few failures related to this PR. @codedsun Please see the report [here](https://mattermost-cypress-report.s3.amazonaws.com/198266-017b1b1-onprem-ent-webapp-pr-10099-mattermost/mm-ee-test017b1b1/mochawesome.html) and let us know if you need help addressing these

**Review by jgilliam17 — CHANGES_REQUESTED** [review]
@codedsun Let me know if you have any questions about addressing E2E failures
See comment here: https://github.com/mattermost/mattermost-webapp/pull/10099#issuecomment-1136650269

**Comment by AshishDhama** [comment]
closing in favour of https://github.com/mattermost/mattermost-webapp/pull/11107

**Pull request closed without merging**